server/src/server.ts

```diff
@@ -135,6 +145,17 @@ function validateTextDocument(textDocument: TextDocument): void {
   }
 }
 
+connection.onDefinition((pos: TextDocumentPositionParams) => {
+  const doc: TextDocument = documents.get(pos.textDocument.uri);
+  const linePrefix = doc.getText().substr(0, doc.offsetAt(pos.position));
+  const lastToken = linePrefix.match(/\b([^:\.\s]+)\b$/g);
+  if (lastToken === null || lastToken.length === 0) {

```

Comment:
It's confusing to have an array called something like “lastFoo” since it sounds like there can be only one such thing. And indeed you seem to be expected at most one element. But that still sounds weird and unnatural.

Reply:
I will rename this one.